src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -1,13 +1,29 @@
-const embedUrl = ({ type, requestUrl, isAmp = false }) => {
-  const avRoute = 'ws/av-embeds';
-  const baseUrl = process.env.SIMORGH_EMBEDS_BASE_URL;
-  const parts = [baseUrl, avRoute, type, requestUrl];
+import isLive from '#lib/utilities/envHelper';
+import hasLiveOverride from '#lib/utilities/rendererEnvHelper';
+import nodeLogger from '#lib/logger.node';
+
+const logger = nodeLogger(__filename);
+
+const AV_ROUTE = 'ws/av-embeds';
+
+const getBaseUrl = url => {
+  if (hasLiveOverride(url) || isLive()) {
```

Comment:
Just to prevent the code running on live:
```suggestion
  if (isLive() || hasLiveOverride(url)) {
```